src/database/queries/sessions.js

```diff
@@ -259,6 +259,7 @@ exports.findAllSessions = async (page, limit, search, filters) => {
 				'image',
 				'created_at',
 				'meeting_info',
+				'created_by',
```

Comment:
@nevil-mathew we need mentor_name right ?

Reply:
we're not using this function for the list apis @rakeshSgr